### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gargarchit**
I want to merge this change because...

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for 8ec7260a440f5ba85c55ca59310dd267aa8da137 (gargarchit/saleor @ patch-1)
Base comparison is 6d5ce07e9f8faa3c0f4b8789cb4945157cb59940.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         14	         14	              0
  complete checkout                          	          6	          6	              0
  create checkout                            	         48	         48	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	         13	         13	              2

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: README.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ Saleor is a rapidly-growing open source e-commerce platform that has served high
 - **GraphQL API**: Access all data from any web or mobile client using the latest technology
 - **Headless commerce**: Build mobile apps, customize storefronts and externalize processes
 - **UX and UI**: Designed for a user experience that rivals even the top commercial platforms
-- **Dashboard**: Administrators have total control of users, processes and products
-- **Orders**: A comprehensive system for orders, dispatch and refunds
+- **Dashboard**: Administrators have total control of users, processes, and products
+- **Orders**: A comprehensive system for orders, dispatch, and refunds
 - **Cart**: Advanced payment and tax options, with full control over discounts and promotions
-- **Payments**: Flexible API architecture allows integration of any payment method. Comes with Braintree support out of the box.
+- **Payments**: Flexible API architecture allows integration of any payment method. It comes with Braintree's support out of the box.
```
Comment (timcla) [inline README.md]: remove 's. Was correct before

Comment (gargarchit) [reply README.md]: Done @timcla

**Comment by gargarchit** [comment]
@NyanKiyoshi @timcla I have update the changes

**Pull request merged**